client/executor/src/integration_tests/mod.rs

```diff
@@ -803,3 +803,30 @@ fn unreachable_intrinsic(wasm_method: WasmExecutionMethod) {
 		error => panic!("unexpected error: {:?}", error),
 	}
 }
+
+test_wasm_execution!(panic_in_host_function);
+fn panic_in_host_function(wasm_method: WasmExecutionMethod) {
```

Comment:
Can we not merge this and the other test above? Both are doing the same?

Reply:
I've left them as-is since in theory they're supposed to be testing something slightly different, but I don't have a strong opinion either way. I can merge them if you want.

Reply:
Okay, merged them together.